src/NzbDrone.Core/Indexers/Definitions/Gazelle/GazelleParser.cs

```diff
@@ -60,7 +60,7 @@ public virtual IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
                 foreach (var torrent in result.Torrents)
                 {
                     // skip releases that cannot be used with freeleech tokens when the option is enabled
-                    if (Settings.UseFreeleechToken && !torrent.CanUseToken)
+                    if (Settings.UseFreeleechToken is (int)GazelleUseFreeleechTokens.Preferred or (int)GazelleUseFreeleechTokens.Required && !torrent.CanUseToken)
```

Comment:
I don't think you should skip these releases if use freeleech tokens is set to preferred. To me, "preferred" means I'm happy to see releases that tokens can't be used on too.